templates/application/layouts/application.html

```diff
@@ -0,0 +1,112 @@
+---
+{
+	"noFooterMain": true,
+	"noFooterCorporate": true,
+	"nositesearch": true,
+	"breadcrumbs": false,
+	"nositesearch": true
+}
+---
+
+{%- include variable-core.liquid -%}
+{%- capture page-title -%}
+	{%- if page.title -%}
+		{{ page.title }}
+	{%- else -%}
+		Page untitled
+	{%- endif -%}
+{%- endcapture -%}
+
+{%- capture applicationName -%}
+	{%- if page.applicationName -%}
+		{{ page.applicationName }}
+	{%- elsif site.global.applicationName[i18nText-lang] -%}
+		{{ site.global.applicationName[i18nText-lang] }}
+	{%- else -%}
+		null
+	{%- endif -%}
+{%- endcapture -%}
+
+{%- capture applicationURL -%}
+	{%- if page.applicationURL -%}
+		{{ page.applicationURL }}
+	{%- elsif site.global.applicationURL[i18nText-lang] -%}
+		{{ site.global.applicationURL[i18nText-lang] }}
+	{%- else -%}
+		null
+	{%- endif -%}
+{%- endcapture -%}
+<!DOCTYPE html>
+<html class="no-js" lang="{{ i18nText-lang | default: 'en' }}" dir="{{ i18nText-langDir | default: 'ltr' }}">
+
+<head>
+	<meta charset="utf-8">
+	{% include license.html %}
+	<title>{{ page-title }} - {{ i18nText-siteTitle }}</title>
+	<meta content="width=device-width, initial-scale=1" name="viewport">
+	{% include metadata.html %}
+	{% include resources-inc/head.html %}
+</head>
+
+<body {% if page.pageclass %}class="{{ page.pageclass }}" {% endif %}vocab="http://schema.org/" typeof="WebPage">
+	{% include skiplinks/skiplinks.html %}
+
+	{%- if page.infoBanner -%}
+	<section class="info-banner">
+		<div class="container">
+			<h2>{{ page.infoBanner.title }}</h2>
+			<p>{{ page.infoBanner.message }}</p>
+			<div class="info-banner-actions">
+				{%- if page.infoBanner.link -%}
+					{%- if page.infoBanner.link.external -%}
```

Comment:
I am not sure about that property name. Having an external link don't means we should enforce the exit-script plugin.

What do you think if we just don't add support for opening the page in a popup + no support for exit script. But we keep the icon if the link was marked to be external.